### PR TITLE
Add check for SQ version to fill the Combobox

### LINKED ISSUE #5030: Add check for SQ version to fill the Combobox.
SC Or SQ version > 10.3 -> Accept
SQ Version <= 10.3 ->  Won't Fix

`IConnectedModeFeaturesConfiguration` could be a good place to put this logic

### PR DESCRIPTION
**Pull request by ugras-ergun-sonarsource**
Fixes #5030

### CODE REVIEW

File: src/Core/Configuration/ConnectedModeFeaturesConfiguration.cs [hunk 1]
```diff
@@ -30,15 +30,22 @@ namespace SonarLint.VisualStudio.Core.Configuration
     public interface IConnectedModeFeaturesConfiguration
     {
         /// <summary>
-        /// Indicates whether Local Hotspot Analysis is supported in the current Connected Mode state  
+        /// Indicates whether Local Hotspot Analysis is supported in the current Connected Mode state
         /// </summary>
-        /// <returns>True if connected to SCloud or SQube 9.7 and above, False otherwise</returns>
```
Comment (georgii-borovinskikh-sonarsource) [inline src/Core/Configuration/ConnectedModeFeaturesConfiguration.cs:35]: why did you remove this?

Comment (ugras-ergun-sonarsource) [reply src/Core/Configuration/ConnectedModeFeaturesConfiguration.cs:35]: I didn't mean to I think instead of copy paste I made cut paste by mistake.
